README.md

```diff
@@ -44,6 +44,29 @@ custom:
     - SomeOtherKeyThatAssignsRandomValue
 ```
 
+### Specifying usage plan (To put both keys in the same usage plan)
+
+```yaml
+custom:
+  apiKeys:
+    - name: SomeKey
+      usagePlan: name-of-same-usage-plan
```

Comment:
@christopherchubb - rather than keeping `usagePlan` as string in some case and object in other can we make the config as

```
usagePlan: (optional)
        name: "name-of-second-plan" (required if usagePlan is specified. rest of the fields are optional)
        description: "Description of second plan"
        quota:
          limit: 2000
          period: DAY
        throttle:
          burstLimit: 100
          rateLimit: 20
```

This will make sure that config is consistent.
thoughts?